doc/source/api.rst

```diff
@@ -845,12 +863,16 @@ Binary operator functions
 
    DataFrame.add
    DataFrame.sub
+   DataFrame.subtract
```

Comment:
Thoughts on these aliases? IMO, since there methods we should document them. But I can also put them in our "hidden" block down below.

Reply:
We should document them in the sense that they have a docstring and will end up in the class docstring page of DataFrame in the full list of members, but I don't think that should mean we should include them all in this place.